### PR TITLE
Create new file.txt

### PR DESCRIPTION
**Pull request by EvgenyRazumov**

### CODE REVIEW

File: new file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello World!
```
Comment (igor-tatarnikov) [inline new file.txt]: World -> world

**Review by igor-tatarnikov — CHANGES_REQUESTED** [review]
1

**Comment by EvgenyRazumov** [comment]
@igor-tatarnikov,
please, review again

**Review by igor-tatarnikov — APPROVED** [review]
done

**Pull request merged**